test/language/expressions/exponentiation/applying-the-exp-operator_A3.js

```diff
@@ -0,0 +1,28 @@
+// Copyright 2016 Rick Waldron.  All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+id: sec-applying-the-exp-operator
+description: >
+    Checking if (base ** exponent) equals to 1, where base is number or NaN
+    and exponent is -0
+---*/
+
+
+var exponent = +0;
```

Comment:
Looks like `test/built-ins/Math/pow/applying-the-exp-operator_A3.js` needs to be updated too.